### PR TITLE
enable tests for real

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/login_requiring/login.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ $order: 1
 description: 'The first time you land on the page, you can see 2 comments and a login button. If you look for the login button in the code, you will find ...'
 ---
 
-The first time you land on the [page]({{g.doc('/content/amp-dev/documentation/examples/previews/Comment_Section.html', locale=doc.locale).url.path}}), you can see 2 comments and a login button.
+The first time you land on the [page]({{g.doc(/documentation/examples/interactivity-dynamic-content/comment_section/preview/?format=websites).url.path}}), you can see 2 comments and a login button.
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/login_requiring/login.md]: This link looks strange know. It's mixing proper URLs with the `g.doc` syntax.

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/login_requiring/summary.md [hunk 2]
```diff
@@ -10,4 +10,4 @@ To learn more, visit the following resources:
 
 - [`amp-access`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-access.md', locale=doc.locale).url.path}})
 - [`amp-form`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-form.md', locale=doc.locale).url.path}})
-- ["Comment Section" Example]({{g.doc('/content/amp-dev/documentation/examples/documentation/Comment_Section.html', locale=doc.locale).url.path}})
+- ["Comment Section" Example]({{g.doc('/content/amp-dev/documentation//examples/documentation/Comment_Section.html', locale=doc.locale).url.path}})
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/login_requiring/summary.md]: Double slash, but will probably work as Python's `path.*` methods handle them gracefully. Applies to some more.

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/media_iframes_3p/iframes.md [hunk 3]
```diff
@@ -86,4 +86,4 @@ Renders as:
 
 ## Examples
 
-You can find more advanced [`amp-iframe`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-iframe.md', locale=doc.locale).url.path}}) examples in [AMP By Example]({{g.doc('/content/amp-dev/documentation/examples/documentation/amp-iframe.html', locale=doc.locale).url.path}}).
+You can find more advanced [`amp-iframe`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-iframe.md', locale=doc.locale).url.path}}) examples in [AMP By Example]({{g.doc('/content/amp-dev/documentation/components/reference/amp-iframe.md', locale=doc.locale).url.path}}).
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/media_iframes_3p/iframes.md]: The Reference Checker mistakenly rewrote the links from an example document to the component.

File: pages/content/amp-dev/documentation/components/index.html [hunk 4]
```diff
@@ -1,100 +0,0 @@
----
```
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/components/index.html]: huh! restored.

**Comment by sebastianbenz** [comment]
Not sure what happened here. Reverted all doc and sample changes. PTAL

**Pull request merged**